### ISSUE TITLE
chipTAN QR

### ISSUE THREAD

**Issue by pc-coholic** [raphaelm/python-fints#57]
The Sparkassen have discovered, that waiting to transmit the chipTAN-data via flickercode takes bloody ages at times and came up with something new and revolutionary: a QR-code!

As far as I can tell, the same data that goes into the flickercode is just put inside a QR-code for scanning convenience.

As of now, it's only used at select Sparkassen, with a global rollout planned for 2019.

I'll have a Reiner-device that I can provide for testing.

flickercode to QR plugins:
https://chrome.google.com/webstore/detail/reiner-sct-qr-code-plugin/mnmcinkgfngjalhihpfidcklaioaldko
https://addons.mozilla.org/de/firefox/addon/reiner-sct-qr-code-plugin/

chipTAN QR devices:
https://shop.reiner-sct.com/tan-generatoren-fuer-sicheres-online-banking/tanJack-QR
https://www.sparkassen-shop.de/home/shop/chiptan-qr,570/

**Comment by raphaelm** [comment]
I *think* current `master` already solves that thanks to Henryk: https://github.com/raphaelm/python-fints/pull/54

Feel free to test and report back :)

**Comment by pc-coholic** [comment]
Yes, seems like it... I didn't see that as I just checked the docs... Sorry for the noise. Will test and report back.

**Comment by raphaelm** [comment]
Feel free to open a docs PR (scnr)
